packages/compass-data-modeling/src/components/diagram-editor-toolbar.tsx

```diff
@@ -58,6 +70,20 @@ export const DiagramEditorToolbar: React.FunctionComponent<{
       data-testid="diagram-editor-toolbar"
     >
       <div className={toolbarGroupStyles}>
+        <Tooltip
+          trigger={
+            <IconButton
+              aria-label="Add Relationship"
```

Comment:
(or maybe even more contextual based on `isInRelationshipDrawingMode` value)

```suggestion
              aria-label="Toggle relationship mode"
```